packages/client/api/package.json

```diff
@@ -1,36 +1,36 @@
 {
-  "name": "@fluencelabs/js-client.api",
-  "version": "0.11.6",
-  "description": "JS Client API",
-  "engines": {
-    "node": ">=10",
-    "pnpm": ">=3"
-  },
-  "type": "module",
-  "source": "src/index.js",
-  "main": "dist/index.js",
-  "umd:main": "dist/index.umd.js",
-  "module": "dist/index.mjs",
-  "exports": {
-    "types": "./dist/index.d.ts",
-    "require": "./dist/index.js",
-    "default": "./dist/index.modern.mjs"
-  },
-  "types": "dist/index.d.ts",
-  "typings": "dist/index.d.ts",
-  "scripts": {
-    "build": "microbundle --generateTypes false src/index.ts && dts-bundle-generator --config dtsbundle.cjs"
-  },
-  "repository": "https://github.com/fluencelabs/fluence-js",
-  "author": "Fluence Labs",
-  "license": "Apache-2.0",
-  "dependencies": {
-    "@fluencelabs/interfaces": "0.7.5"
-  },
-  "devDependencies": {
-    "@fluencelabs/avm": "0.35.4",
-    "@fluencelabs/marine-js": "0.3.45",
-    "microbundle": "0.15.1",
-    "dts-bundle-generator": "7.2.0"
-  }
+    "name": "@fluencelabs/js-client.api",
+    "version": "0.11.6",
+    "description": "JS Client API",
+    "engines": {
+        "node": ">=10",
+        "pnpm": ">=3"
+    },
+    "type": "module",
+    "source": "src/index.js",
+    "main": "dist/index.js",
+    "umd:main": "dist/index.umd.js",
+    "module": "dist/index.mjs",
+    "exports": {
+        "types": "./dist/index.d.ts",
+        "require": "./dist/index.js",
+        "default": "./dist/index.modern.mjs"
+    },
+    "types": "dist/index.d.ts",
+    "typings": "dist/index.d.ts",
+    "scripts": {
+        "build": "microbundle --generateTypes false src/index.ts && dts-bundle-generator --config dtsbundle.cjs"
+    },
+    "repository": "https://github.com/fluencelabs/fluence-js",
+    "author": "Fluence Labs",
+    "license": "Apache-2.0",
+    "dependencies": {
+        "@fluencelabs/interfaces": "0.7.5"
+    },
+    "devDependencies": {
+        "@fluencelabs/avm": "https://github.com/fluencelabs/js-client/raw/feat/add-signatures/avm-0.39.1-feat-VM-276-aquavm-keypair-e2816f6-1588-1.0.tgz",
```

Comment:
Ohmy! Didn't know you can do that :D

You can also use npm snapshot from here https://github.com/fluencelabs/aquavm/actions/runs/5326142611?pr=593#summary-14414451030